SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -5764,9 +5860,194 @@ void CEulerSolver::SetActDisk_BCThrust(CGeometry *geometry, CSolver **solver_con
   Factor = (0.5*RefDensity*RefArea*RefVel2);
   Ref = config->GetDensity_Ref() * config->GetVelocity_Ref() * config->GetVelocity_Ref() * 1.0 * 1.0;
 
+  su2double Dens_FreeStream = config->GetDensity_FreeStream();
+  su2double *Vel_FreeStream = config->GetVelocity_FreeStream();
+
+  /*--- Variable load distribution is in input file. ---*/
+  /*--- Input file provides force coefficients distributions along disk radius. Initialization
```

Comment:
Ah just noticed this block of code is in SetActDisk_BCThrust, can it be moved to its own private member function? We should try to keep functions on the smallish side (e.g. fits in one screen).

Reply:
Did it. I moved it in another function named `ReadActDisk_InputFile`. Now the `SetActDisk_BCThrust` is more readable.